MeasureAuthoringTool/mat-drupal
# Init base theme

## Description
Adds custom theme from Emulsify base theme. Sets new theme as base theme, export config to capture these settings. 

** You will need node v11.15 to build the theme dependencies. Current LTS version throws compilation errors **

## To Test
1. Update composer dependencies `composer install`
2. Install theme dependencies `cd html/themes/custom/emat && npm install`
3. Build the theme `npm start`.
4. Import config into Drupal install (assumes you already have the Drupal install running) `docker-compose exec drupal drush cim -y`
5. Clear caches `docker-compose exec drupal drush cr`
6. Visit site, you should see a very generic-looking page, something like this:

<img width="760" alt="Screen Shot 2020-02-13 at 11 05 48 AM" src="https://user-images.githubusercontent.com/1919263/74459685-5f839d80-4e51-11ea-93b3-a1fa3d08d784.png">
